lib/postcode_validation/use_case/format_validators/be_postcode_validator.rb

```diff
@@ -0,0 +1,6 @@
+class BEPostcodeValidator
```

Comment:
Yeah this is a slight variation on the [template method pattern](https://en.wikipedia.org/wiki/Template_method_pattern)

Here we see the same logic shared by 4 classes in an abstract class, and the "template method" - the constant `REGEX` - supplied to it by the concrete "BEPostcodeValidator"